### PR TITLE
fix(org:usage): Support for package bundles and unlimited SaaS

### PR DESCRIPTION
**Pull request by cb1kenobi**
JIRA: https://jira.axway.com/browse/APIGOV-19513

### CODE REVIEW

File: packages/axway-cli-oum/src/org/usage.js [hunk 1]
```diff
@@ -51,41 +99,90 @@ export default {
...
+					const env = metric.envs[i];
+					table.push([
+						`  ${i + 1 === len ? '└─' : '├─'} ${env.name} ${env.production ? gray('Production') : ''}`,
+						`${highlight(format(env.value).padStart(bundleValuePadding))} Transactions${env.tokens && ratio !== 1 ? highlight(` x ${(ratio / 100).toFixed(1)}`) : ''}`,
+						env.tokens ? { content: `${highlight(format(env.tokens))} ${bundle.units}`, hAlign: 'right' } : ''
+					]);
+				}
+			}
+			if (table.length) {
+				console.log();
```
Comment (agrakhov) [inline packages/axway-cli-oum/src/org/usage.js:148]: minor - maybe combine in one log('\n', table.toString())?

File: packages/axway-cli-oum/src/org/usage.js [hunk 2]
```diff
@@ -51,41 +99,90 @@ export default {
...
+							`  ${i + 1 === len ? '└─' : '├─'} ${name} ${production ? gray('Production') : ''}`,
+							`${highlight(formatted.padStart(saasValuePadding))} ${unit}`,
+							''
+						]);
+					}
+				}
+			}
+		}
+		if (table.length) {
+			console.log(table.toString());
```
Comment (agrakhov) [inline packages/axway-cli-oum/src/org/usage.js:185]: is a new line also needed here?

File: test/helpers/platform-routes.js [hunk 3]
```diff
@@ -189,29 +209,47 @@ export function createPlatformRoutes(server, opts = {}) {
...
-			}
+			const maxEntitlement = 9999999999999;
+			let result = null;
+
+			if (usage) {
+				// SaaS
+				const SaaS = {};
+
+				for (const [ type, meta ] of Object.entries(types)) {
+					const quota = org.guid === '3000' ? maxEntitlement : usage.quotas[type];
```
Comment (agrakhov) [inline test/helpers/platform-routes.js:220]: Add a file const with describing name, maybe?

File: packages/axway-cli-oum/src/org/usage.js [hunk 4]
```diff
@@ -33,15 +35,61 @@ export default {
...
+
+		// pre-determine the bundle metric environments
+		if (bundle) {
+			for (const metric of Object.values(bundle.metrics)) {
+				metric.envs = Object.entries(metric.envs).map(([ guid, stats ]) => {
+					bundleValuePadding = Math.max(bundleValuePadding, String(format(stats.value)).length);
+					return {
+						guid,
+						name: orgEnvs.find(e => e.guid === guid)?.name || guid,
+						...stats
```
Comment (agrakhov) [inline packages/axway-cli-oum/src/org/usage.js:57]: Any chances that stats might have a guid or name?

**Review by agrakhov — APPROVED** [review]

File: packages/axway-cli-oum/src/org/usage.js (hunk 4 above)
Comment (cb1kenobi) [reply packages/axway-cli-oum/src/org/usage.js]: Good point, better safe than sorry!

File: packages/axway-cli-oum/src/org/usage.js (hunk 2 above)
Comment (cb1kenobi) [reply packages/axway-cli-oum/src/org/usage.js:185]: Nope, we have the line break on line 161:
```js
table.push([ `\n${bold(label)}` ]);
```

File: test/helpers/platform-routes.js (hunk 3 above)
Comment (cb1kenobi) [reply test/helpers/platform-routes.js:221]: Fixed!

**Pull request merged**